### ISSUE TITLE
Alternative to MergeStructures for implementing docking view

### ISSUE THREAD

**Issue by QDMarkMan** [molstar/molstar#712]
Description:

In the current implementation of docking view, the MergeStructures built-in transform is being used. However, some users have reported concerns about the performance and functionality of this approach. It's not very intuitive for users to use.

![20230207135927_rec_](https://user-images.githubusercontent.com/18100675/217161059-826a441b-2aaa-48ff-966b-9bdb4b0e1d01.gif)

We would like to request the development of an alternative solution that does not use MergeStructures and can effectively implement the docking  view. 

The ideal solution should maintain the current level of functionality and support for multiple input structures. It should also be easy to integrate into the existing codebase and not cause any compatibility issues with other features or transforms.

We appreciate your time and efforts in addressing this issue and look forward to seeing a solution.

[Code location](https://github.com/molstar/molstar/blob/master/src/apps/docking-viewer/index.ts#L185)

**Comment by arose** [comment]
The easiest way for you is to create your own UI instead of using the state tree UI. A more general solution could be to support an easy way to map the state tree to a custom UI, if you want to explore that a PR is appreciated.

**Comment by QDMarkMan** [comment]
I'd be happy to explore a PR, but is there some more detailed documentation about `State` and `MergeStructures`?

**Comment by arose** [comment]
There is mostly only the source code. Some notes in https://github.com/molstar/molstar/tree/master/docs/state and in answers to issues.

**Comment by QDMarkMan** [comment]
> 

Thank you for your previous response, but I still have a question. I removed the `MergeStructures`  component, but now neither the `surroundings` nor the `interactions` are being rendered. I suspect that the issue may be related to the data source. Can you provide insight on the data requirements for drawing the `surroundings` and `interactions`? Your help would be greatly appreciated. Thank you in advance!

**Comment by arose** [comment]
Please see if #1463 solves it for you.

**Comment by QDMarkMan** [comment]
The new features look great! Do you have any examples or documents that I could refer to as a reference?

**Comment by dsehnal** [comment]
Yes, there is new `interactions` example that showcases the functionality.
